compute/admin_guide/compliance/detect_secrets.adoc

```diff
@@ -55,10 +56,87 @@ image::detect_secrets_762507.png[width=800]
 Set the action to *ALERT* instead of *BLOCK*, then go to *Monitor > Compliance* after running the image.
 Click on the image under *Images* tab.
 
-image::detect_secrets_762508.png[width=800]
+==== Compliance Check ID 597
 
+This check detects sensitive information provided in environment variables of container.
 
-[.section]
-=== Compliance check ID 597
+[#agentless-secret-detection]
+=== Agentless Secret Detection
 
-This check detects sensitive information provided in environment variables of container.
+Agentless scanning detects sensitive information inside your hosts and container images.
+
+==== Compliance Check ID 456
+
+This check detects secrets inside your hosts' filesystem.
+By default, Prisma Cloud alerts you that a secret was detected in a host.
+Select *Monitor > Compliance > Compliance Explorer* and filter the results using `secret` as a keyword to see the impacted hosts.
+
+==== Compliance Check ID 457
+
+This check detects secrets inside your container images' filesystem.
+By default, Prisma Cloud alerts you that a secret was detected in a container image.
+Select *Monitor > Compliance > Compliance Explorer* and filter the results using `secret` as a keyword to see the impacted container images.
+
+[#detected-secrets]
+==== Detected Secrets
+
+When agentless scanning detects a secret in a host or container image, only the following information is sent to the Prisma Cloud console.
+
+* A redacted snippet of the secret with a length of less than 10 characters.
+* The filename where the secret was detected.
+* The line where the secret was detected.
```

Comment:
The line and offset in the file where the secret was detected in the form of `line:offset`.